app/core/application.js

```diff
@@ -109,6 +109,7 @@ const Application = {
     }
     this.remindPlayerToTakeBreaks()
     if (utils.isCodeCombat) { userUtils.provisionPremium() }
+    userUtils.provisionEsports()
```

Comment:
why would we add a one more db call? We could have just modified provisionPremium to work for all users instead of non-libraries. This is an expensive addition since on each refresh, you are adding a extra db call